schedule/yast/textmode/ha_textmode_skip_registration_s390x.yaml

```diff
@@ -0,0 +1,34 @@
+name: ha_textmode_skip_registration
+description: >
+  Installation SLES + HA in textmode which selects System Role 'Text Mode'.
+vars:
+  VIDEOMODE: text
+  DESKTOP: textmode
+  SCC_REGISTER: 'none'
+  SCC_ADDONS: ha
+  ADDONS: all-packages
```

Comment:
with libyui-rest-api we don't need most of this settings above, `SCC_REGISTER` or `ADDONS`.
The rest could go to job group yaml.